main.go

```diff
@@ -1,26 +1,26 @@
 package main
 
 import (
-	"fmt"
 	"github.com/gofiber/fiber"
-	"learning_go/book"
+	"github.com/gofiber/fiber/v2/middleware/recover"
 	"learning_go/database"
+	"learning_go/routes"
 	"log"
 )
 
 func configureRoutes(app *fiber.App) {
-	app.Get("api/v1/books", book.GetBooks)
-	app.Get("api/v1/books/:id", book.GetBook)
-	app.Post("api/v1/books", book.AddBook)
-	app.Delete("api/v1/books/:id", book.DeleteBook)
+	app.Get("api/v1/books", routes.GetBooks)
+	app.Get("api/v1/books/:id", routes.GetBook)
+	app.Post("api/v1/books", routes.AddBook)
+	app.Delete("api/v1/books/:id", routes.DeleteBook)
 }
 
 func main() {
+	database.InitDatabase()
```

Comment:
I'd probably do a `defer` call here to close the database after this line so when the program terminates it will cleanly close its connection to the database. Especially with a file-based DB like SQLite you wouldn't want to corrupt anything if it doesn't get closed down properly.